Applications/spire/source/book_view/quote_panel_indicator_widget.cpp

```diff
@@ -0,0 +1,47 @@
+#include "spire/book_view/quote_panel_indicator_widget.hpp"
+#include <QPainter>
+
+using namespace Spire;
+using MS = std::chrono::milliseconds;
```

Comment:
Remove the ```using```, it's one thing if you reuse these in many many places, but if it's just used in like a couple of places then it's not worth it.